docs/plots/backends.rst

```diff
@@ -0,0 +1,362 @@
+************************
+Plotting backend details
+************************
+
+.. warning::
+   The plotting backend API is currently re-designed and details might change in the near future.
+
+Plotting in Sherpa is done through *plotting backends*. A *backend* is an
+external package that performs the actual plotting function such as :term:`matplotlib`, i.e. it creates a
+canvas and puts color on paper or on the screen. The Sherpa plotting classes are
+written in a backend-independent way; they work with any plotting backend that
+is supported by Sherpa. We also use the term *backend* to refer to Sherpa
+classes that connect the Sherpa plot objects with the plotting backend by
+translating Sherpa options into backend-specific commands. The details of that
+are explained below, but, in short, a backend class would translate the sherpa
+backend-independent "plot a line in red from here to there" into backend
+specific ``plt.plot(x,y, linecolor='r')`` (if the backend is :term:`matplotlib`).
+
```

Comment:
I think this needs re-writing as you point out in the middle that we actually have two concepts of `backend` - the code we have written to interface to the backend, and the actual backend. I don't have the energy to write a suggestion just let but let me come back to it (unless anyone else has a strong desire to write something)

Reply:
I'll reword that.